src/ui/components/abstract/ScriptRegion.mjs

```diff
@@ -0,0 +1,17 @@
+import { multipleClasses } from '../../../utils';
+
+import LayoutFrame from './LayoutFrame';
+import ScriptObject from './ScriptObject';
+
+class ScriptRegion extends multipleClasses(ScriptObject, LayoutFrame) {
```

Comment:
We'll still have access to local properties on the superclasses if we do this, right?

Reply:
Yes 👍